streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -72,7 +72,7 @@
  * }</pre>
  */
 @InterfaceStability.Evolving
-public class Stores {
+public final class Stores {
```

Comment:
Is this intentional? I'm concerned someone out there in the wild may have extended this class for their customized needs.

Reply:
There was a warning. Note that `Stores` has only static methods, thus it does not make sense to inherit from it. I can still revert it. Don't feel strong about it.